docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -113,8 +115,8 @@ when using security-sensitive API modules in native applications:
 | `http://tizen.org/privilege/imemanager` | public |  | The application can manage installed input methods. |
 | `http://tizen.org/privilege/inputgenerator` | platform |  | The application can simulate keys being pressed and touch interactions with the screen. |
 | `http://tizen.org/privilege/keygrab` | platform |  | The application can read actions involving special keys, such as the volume keys on this or other devices (for example, TV remote controls), even when it is running in the background. |
-| `http://tizen.org/privilege/keymanager` | public |  | The application can save keys, certificates, and data to, and retrieve and delete them from, password-protected storage. Checking the statuses of certificates while connected to a mobile network may result in additional charges depending on user's payment plan. |
-| `http://tizen.org/privilege/keymanager.admin` | platform |  | The application can lock and unlock password-protected storage, and manage password changes for it. |
+| `http://tizen.org/privilege/keymanager` | public |  | The application can save keys, certificates, and data to, and retrieve and delete them from, password-protected storage. Checking the statuses of certificates while connected to a mobile network may result in additional charges depending on user's payment plan. Deprecated since 3.0. |
+| `http://tizen.org/privilege/keymanager.admin` | platform |  | The application can lock and unlock password-protected storage, and manage password changes for it. Deprecated since 3.0. |
```

Comment:
Just a question. `http://tizen.org/privilege/keymanager.admin` is "deprecated since 2.4" for mobile, and "deprecated since 3.0" for wearable. 

However, as we don't separate mobile/wearable info in this table, only "deprecated since 3.0" sentence is added.
I'm just thinking whether it is acceptable to list up the **later version** among mobile and wearable.

Would you give me your opinion?

Same doubt to web privileges. (htttp://tizen.org/privilege/bluetooth.admin and http://tizen.org/privilege/bluetooth.gap)

Reply:
As we don't separate profiles, we decide to provide privileges from when it first added and last deprecated from platform. There was no wearable profile in platform 2.4, we had to deprecate it at 3.0 for wearable. So I think it would be acceptable.